README.md

```diff
@@ -10,7 +10,7 @@ Full documentation is available on the [JHipster documentation for microservices
 
 Click this button to deploy your own instance of the registry:
 
-[![Deploy to Heroku](https://camo.githubusercontent.com/c0824806f5221ebb7d25e559568582dd39dd1170/68747470733a2f2f7777772e6865726f6b7563646e2e636f6d2f6465706c6f792f627574746f6e2e706e67)](https://dashboard.heroku.com/new?&template=https%3A%2F%2Fgithub.com%2Fjhipster%2Fjhipster-registry)
+[![Deploy to Heroku](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)
```

Comment:
The page at https://heroku.com/deploy uses the HTTP referrer, which is best because it can even pick up the branch or fork this is coming from. The only time you need to put the template explicitly is when you're linking from another page (like in the jhipster docs).